src/communicate/main.py

```diff
@@ -1661,15 +1661,15 @@ def record_search_whiteness(user: int, search_id: int, seach_following_flag) ->
     logging.info(f'{user_id=}')
     # when user pushed INLINE BUTTON for topic following
     if user_callback and user_callback["action"] == "search_follow_mode":
-        ikb = callback_query.message.reply_markup
+        ikb = callback_query.message.reply_markup.inline_keyboard
         logging.info(f'{ikb=}')
         i=-1
         for row in ikb:
             i=i+1
             if int(row[0]['callback_data']['hash'])==int(user_callback['hash']):##if this button was pushed
```

Comment:
do we want to have a checker if/else or try/except if the row[0] does not have ['callback_data'] or ['hash'].
and then in the code ['text'] and length of ['text'] > 1 (in order to do [2:] )

so could you pls consider what to do if the incoming `row` comes in a corrupted way. thanks